js/scenes/Signup/SignupContainer.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { FirebaseAuth } from '../../config/firebase';
+import { signUp } from '../../config/helpers';
+import Signup from './Signup';
+import { 
+  updateEmailField, 
+  updatePasswordField,
+  updateFullnameField
+} from '../../redux/modules/user-forms';
+
+class SignupContainer extends Component {
+
+  handleEmail = (event) => {
+    this.props.dispatch(updateEmailField(event));
+  }
+
+  handlePassword = (event) => {
+    this.props.dispatch(updatePasswordField(event));
+  }
+
+  handleFullname = (event) => {
+    this.props.dispatch(updateFullnameField(event));
+  }
+
+  signupHandler = () => {    
+    if (this.props.emailField.length && this.props.passwordField.length && this.props.fullnameField.length) {
+      const user = {
+        email: this.props.emailField,
+        password: this.props.passwordField,
+        name: this.props.fullnameField
+      }
+      signUp(user)
+    }
+  }
+
+
+  render() {
+    return (
+      <Signup
+        emailField={this.props.emailField}
+        passwordField={this.props.passwordField}
+        fullnameField={this.props.fullnameField}
+
+        signupHandler={() => {
```

Comment:
These functions can be written like this: ```signupHandler={this.signupHandler}```. No need to use the anonymous arrow function. The event object will be passed in as you expect.